src/helix-ui/styles/components/tooltips.less

```diff
@@ -1,6 +1,6 @@
 @import (reference) 'mixins';
 
-hx-tooltip {
+hx-tooltip:extend(.hxComponent) {
   #Positionable.base();
 
   color: @gray-900;
```

Comment:
Please remove `color` property. It's already inherited from `.hxComponent` styles.